TTYC.Domain/UserProfile.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace TTYC.Domain
+{
+	public class UserProfile
+	{
+		public Guid UserId { get; set; }
+
+		public string Name { get; set; }
+		public string Surname { get; set; }
+		public string Email { get; set; }
+
+		public User User { get; set; }
```

Comment:
It is a good idea to mark this reference properties with virtual keyword. And keep foreign key Id and reference property close to each other. Check all places